### PR TITLE
Compile cli fix

### PR DESCRIPTION
**Pull request by tbosch**

### CODE REVIEW

File: modules/angular2/src/compiler/metadata_resolver.ts [hunk 1]
```diff
@@ -417,3 +415,19 @@ function calcTemplateBaseUrl(reflector: ReflectorReader, type: any,
 
   return reflector.importUri(type);
 }
+
+// Only fill CompileIdentifierMetadata.runtime if needed...
```
Comment (tbosch) [inline modules/angular2/src/compiler/metadata_resolver.ts]: Remove this comment

File: gulpfile.js [hunk 2]
```diff
@@ -1053,7 +1053,7 @@ gulp.task('!test.compiler_cli.codegen', function(done) {
   try {
     require('./dist/js/cjs/compiler_cli')
         .main("tools/compiler_cli/test")
-        .then(function() { done(); })
+        .then(function() { runTsc('tools/compiler_cli/test', done); })
```
Comment (alexeagle) [inline gulpfile.js]: The bug was that compiler_CLI skipped the type check step?

Comment (tbosch) [inline gulpfile.js]: yes. At least when running the `gulp test.compiler_cli` target.

**Comment by alexeagle** [comment]
LGTM thanks

**Pull request closed without merging**